developer/build_plugin.py

```diff
@@ -30,7 +30,7 @@
 
 # sgtk imports
 from tank import LogManager
-from tank.util import filesystem, re
+from tank.util import filesystem, sgre as re
 from tank.errors import TankError
 from tank.descriptor import Descriptor, descriptor_uri_to_dict, descriptor_dict_to_uri
 from tank.descriptor import create_descriptor, is_descriptor_version_missing
```

Comment:
The pre-commit hook flagged these files as having an executable bit and missing a she-bang, so I fixed that mistake and removed the bit.